src/carnot/funcs/builtins/json_ops.h

```diff
@@ -228,6 +228,49 @@ class PluckArrayUDF : public udf::ScalarUDF {
   }
 };
 
+class SplitUDF : public udf::ScalarUDF {
+ public:
+  StringValue Exec(FunctionContext*, StringValue in, StringValue delimiter) {
+    rapidjson::StringBuffer sb;
+    rapidjson::Writer<rapidjson::StringBuffer> writer(sb);
+    writer.StartArray();
+
+    std::string_view s(in.data(), in.size());
+    size_t idx = 0;
+    while (idx < s.size()) {
```

Comment:
Can we just use `absl::StrSplit` instead? Then write the results to the json array.

Reply:
Yep, didn't realize it handled using `string_view`s out of the box. Updated.